app/config/AppConfig.scala

```diff
@@ -62,4 +62,6 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: CTCServicesCon
 
   val internalAuthToken: String = config.get[String]("internal-auth.token")
 
+  val enablePhase5: Boolean = config.get[Boolean]("enable-phase-5")
```

Comment:
This is a bad config name, as it's not about enabling phase 5, it's about disabling phase 4.

Reply:
will update the name to `disable-phase-4`